src/main/kotlin/app/revanced/patches/tiktok/feedfilter/patch/TiktokFeedFilter.kt

```diff
@@ -12,14 +12,13 @@ import app.revanced.patcher.patch.annotations.Patch
 import app.revanced.patcher.patch.impl.BytecodePatch
 import app.revanced.patches.tiktok.feedfilter.annotations.FeedFilterCompatibility
 import app.revanced.patches.tiktok.feedfilter.fingerprints.FeedApiServiceLIZFingerprint
-import app.revanced.patches.tiktok.misc.integrations.patch.TikTokIntegrationsPatch
 import app.revanced.patches.tiktok.misc.settings.fingerprints.SettingsStatusLoadFingerprint
 import app.revanced.patches.tiktok.misc.settings.patch.TikTokSettingsPatch
 import org.jf.dexlib2.Opcode
 import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
 
 @Patch
-@DependsOn([TikTokIntegrationsPatch::class, TikTokSettingsPatch::class])
+@DependsOn([TikTokSettingsPatch::class])
```

Comment:
This patch depends on the integrations, as it references the integrations below in the execute method, why is it removed? Also the patch does not use `TikTokSettingsPatch` anywhere, why is it a dependency?

Reply:
> Also the patch does not use `TikTokSettingsPatch` anywhere, why is it a dependency?
I thought `DependsOn` be used when a patch need another patch to work. And this patch need settings patch to change the filter option.
> This patch depends on the integrations, as it references the integrations below in the execute method, why is it removed?
It depends on `TikTokSettingsPatch` and `TikTokSettingsPatch` depends on `TikTokIntegrationsPatch`

Reply:
While `TikTokSettingsPatch` depends on the integrations, this one does too, hence why it should still remain in the array.